api/src/main/java/io/druid/data/input/impl/SpatialDimensionSchema.java

```diff
@@ -67,15 +96,20 @@ public boolean equals(Object o)
     if (dimName != null ? !dimName.equals(that.dimName) : that.dimName != null) {
```

Comment:
Using two different styles of comparison of nullable objects in one method could raise questions, could you please also replace this with Objects.equals()?